01_Core/week02/submissions/style.css

```diff
@@ -0,0 +1,183 @@
+/* STYLES FOR DROPDOWN MENU IN NAV BAR */
+.dropbtn {
+    padding: 16px;
+    font-size: 16px;
+    border: none;
+  }
+  
+.dropdown {
+    position: relative;
+    display: inline-block;
+  }
+  
+.dropdown-content {
+    display: none;
+    background-color: orange;
+    position: absolute;
+    min-width: 160px;
+    z-index: 1;
+  }
+  
+.dropdown-content a {
+    color: black;
+    padding: 12px 16px;
+    text-decoration: none;
+    display: block;
+  }
+  
+.dropdown-content a:hover {background-color: gray;}
+  
+.dropdown:hover .dropdown-content {display: block;}
+  
+.dropdown:hover .dropbtn {background-color: gray;}
+
+/* STYLES FOR NAV BAR */
+.btn {
+    padding: 16px;
+    font-size: 16px;
+    border: none;
+    text-decoration: none;
+}
+
+.btn:hover {background-color: gray;}
+
+.nav-bar  {
+    font-family: Arial;
+    position:fixed;  
+    z-index:1;
+    width: 100%;
+    height: 50px;
+    text-align: center;
+    vertical-align: middle;
+    line-height: 50px;
+    top: 0px;
+}
+
+/* ABOUT SECTION */
+h1  {
+    color: gray;
+    padding: 60px 0px;
+    font-family: Arial;
```

Comment:
when specifying font-family, you should have fallback fonts in case the user's machine doesn't have the specified one loaded. always end with a generic serif or sans-serif.